account/guides/setupRegistrar.md

```diff
@@ -0,0 +1,147 @@
+```bash
+md5 -s sipauthtest:<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password
+MD5 ("sipauthtest@<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password") = 79bb0e55551e14a2f329a282c7cf145
+```
+
+#### Non-Default Setup (_Domain Specified_)
+##### Generate md5 Hash1 _with_ domain `somewhere.com`
+| composite-username  | : | realm                       | : | password   |
+|---------------------|---|-----------------------------|---|------------|
+| `bob@somewhere.com` | : | `<randomAccountHex>.auth.bandwidth.com` | : | `password` |
+```bash
+md5 -s bob@somewhere.com:<randomAccountHex>.auth.bandwidth.com:password
+MD5 ("bob@somewhere.com:<randomAccountHex>.auth.bandwidth.com:password") = 817d76e91aad032a8c272229f468bfb2
+```
+
+#####  Generate md5 Hash2 _with_ domain `somewhere.com`
+| composite-username  | : | realm                       | realm                       | : | password   |
+|---------------------|---|-----------------------------|-----------------------------|---|------------|
+| `bob@somewhere.com` | : | `<randomAccountHex>.auth.bandwidth.com` | `<randomAccountHex>.auth.bandwidth.com` | : | `password` |
+```bash
+md5 -s bob@somewhere.com:<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password
+MD5 ("bob@somewhere.com:<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password") = 39679d2a73c2e1ea719621bc0d8fdac8
+```
+
+### Add the Newly Created Hash to Sip Credentials {#add-new-credentials}
+After generating the hash value, fill in the Hash1 and Hash2 values to create the SIP credentials.
+
+#### Using the UI  {#using-the-ui}
+Select `Sip Credentials` from the account overview page on the [Bandwidth Dashboard](dashboard.bandwidth.com)
+
+<img src="../../images/sip_credentials.png" style="max-width:95%">
+
+<img src="../../images/create_sip_credentials.png" style="max-width:95%">
+
+#### Using the API
+You can also use the API to create SIP credentials after generating the MD5 hash value. When using the API, Bandwidth will not generate the hashes for you - you must supply your own hash values. You can create multiple SIP credentials within one API call.
+
+```http
+POST https://dashboard.bandwidth.com/api/accounts/{accountId}/sipCredentials
+HTTP/1.1
+Content-Type: application/xml; charset=utf-8
+Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
+
+<SipCredentials>
+    <SipCredential>
+        <UserName>sipauthtest</UserName>
+        <Hash1>fe438bddfc087dda89d29e637f5684ab</Hash1>
+        <Hash1b>79bb0e55551e14a2f329a282c7cf145</Hash1b>
+    </SipCredential>
+    <SipCredential>
+        <UserName>bob</UserName>
+        <Domain>somewhere.com</Domain>
+        <Hash1>817d76e91aad032a8c272229f468bfb2</Hash1>
+        <Hash1b>39679d2a73c2e1ea719621bc0d8fdac8</Hash1b>
+    </SipCredential>
+</SipCredentials>
```

Comment:
Dunno if this really matters since if Realm isn't specified in the SipCredentials payload then the default Realm will be used, but if we wanted to make clear that we can specify a `<Realm>` element in a SIP Credential request it could be added to one of the `<SipCredential>` bodies here.

Reply:
done